### PR TITLE
Additional logging and returns

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: subscriptions/tasks.py [hunk 1]
```diff
@@ -63,22 +63,26 @@ def run(self, subscription_id, **kwargs):
         """
         l = self.get_logger(**kwargs)
 
-        l.info("Loading Subscription")
+        l.info("SNM: Loading Subscription")
```
Comment (imsickofmaps) [inline subscriptions/tasks.py]: Prefixing is not required - it's already in the logs

**Comment by imsickofmaps** [comment]
👍  when prefixing gone

**Comment by imsickofmaps** [comment]
👍  again

**Pull request merged**